package.json

```diff
@@ -57,12 +59,26 @@
   },
   "devDependencies": {
     "coveralls": "~3.0.2",
+    "husky": "~1.1.0",
     "istanbul": "~0.4.5",
     "jshint": "~2.9.6",
+    "lint-staged": "^7.3.0",
     "mocha": "5.2.0",
     "nock": "10.0.1",
+    "prettier": "^1.14.2",
     "should": "13.2.3",
     "timekeeper": "2.1.2",
     "watch": "~1.0.2"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": [
+      "prettier --parser flow --single-quote --trailing-comma es5 --write",
```

Comment:
It would be great to avoid sentece repetition and just use:

```
"npm prettier",
```

However, I've realized that the sentence is not exaclly the same, `--write` vs. `--write **/*.js *.js`... Is there any solution to "factorize" common sentence in just one place to be used in both cases?

Reply:
If there is I haven't found it yet. `lint-staged` is running `prettier` across each individual JavaScript file within the commit, the `npm run prettier` script is a convenience function for running across all files - it shouldn't need to be used much, but I find it useful to format my files locally whilst I'm working.

Reply:
Ok. Pitty.

NTC